client/state/transactions/reducer.js

```diff
@@ -0,0 +1,56 @@
+/** @format */
+
+/**
+ * Internal dependencies
+ */
+
+import { combineReducers, createReducer } from 'state/utils';
+import { TRANSACTION_CREATE_REQUEST, TRANSACTION_CREATE_SUCCESS, TRANSACTION_CREATE_FAILURE } from 'state/action-types';
+
+// Gets set and reset on success / failure
+export const isFetching = createReducer(
+	false, // Default request
+	{
+		[ TRANSACTION_CREATE_REQUEST ]: ( state, action ) => action.request,
```

Comment:
I think we want `isFetching` to be `true` when we dispatch `TRANSACTION_CREATE_REQUEST`.

Reply:
We could throw in a quick test for the reducer items too. There are plenty of examples under `/client/state/{dir}/test`

Something like this would catch any funny business in our expectations for `state.transaction.isFetching`

```
	describe( '#isFetching()', () => {
		test( 'should default to false', () => {
			const state = isFetching( undefined, {} );
			expect( state ).toBe( false );
		} );

		test( 'should be true when we dispatch a request', () => {
			const state = isFetching( undefined, {
				type: ORDER_TRANSACTION_FETCH,
			} );
			expect( state ).toEqual( true );
		} );

    // ... and so on for each state manifestation you need to test
	} );
```